README.md

```diff
@@ -47,6 +47,11 @@ github-contrib -token=<your-token> <org> <github-handle>
     	print version and exit
 ```
 
+Example: get contribution list for `nikhita` against `kubernetes` community.
```

Comment:
Can you add this as a sub-section under Usage?

Reply:
I'd prefer to separate `Example` with `Usage`.

Because the `Example` seems dump from code. 

Do you mean add Example to code?

Reply:
You'd like this? 
```
github-contrib : v0.1.0
USAGE:
github-contrib -token=<your-token> <org> <github-handle>
  -token string
    	Mandatory GitHub API token
  -v	print version and exit (shorthand)
  -version
    	print version and exit
Example: 
# github-contrib -token=<token> kubernetes nikhita
```

Reply:
Revisiting this, I think this is ok as-is. :)